src/pages/scheduler/components/Event.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Heading, HStack, VStack } from '@chakra-ui/react';
+import { EventProps } from 'react-big-calendar';
+
+export const CustomEvent = ({ title, event }: EventProps) => {
+  return (
+    <Flex height="100%" direction="column">
+      <HStack w="100%" bg="#EDF2F7" justifyContent="space-between" p="0.2em">
```

Comment:
could probably move away from hard coding hexes in, even if Chakra doesn't offer the _exact_ same shade. Also, maybe dark mode integration 😉  ?

Reply:
you love dark mode don't you :wink: done.